examples/jsm/loaders/PCDLoader.js

```diff
@@ -97,6 +97,49 @@ class PCDLoader extends Loader {
 
 	}
 
+	/**
+	 * Get dataview value by field type and size.
+	 *
+	 * @param {DataView} dataview - The DataView to read from.
+	 * @param {number} offset - The offset to start reading from.
+	 * @param {'F' | 'U' | 'I'} type - Field type.
+	 * @param {number} size - Field size.
+	 * @returns {number} Field value.
+	 */
+	getDataView(
```

Comment:
I'm unable to edit your PR so you must do it instead. 

Please ensure properly formatting of your changes. Right now, the project's code style is ignored. Please make sure your editor supports ESLint so it automatically honors the ESLint settings of the project.

Besides, `getDataView()` should not be a public method. Please use `_getDataView()` to mark it as private.